person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age, :parent_permission
```

Comment:
- For the `parent_permission` attribute, please keep in mind that it has a default value set to true, so it doesn't need to be read or accessed. So it's more logically to remove it here 🙏

Reply:
thank you!